doc/api/v1/sessions/sign_in_with_valid_password.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resource": "Sessions",
+  "http_method": "POST",
+  "route": "/v1/users/sign_in",
+  "description": "Sign in with valid password",
+  "explanation": null,
+  "parameters": [
+    {
+      "required": true,
+      "name": "email",
+      "description": "Email"
+    },
+    {
+      "required": true,
+      "name": "password",
+      "description": "Password"
+    }
+  ],
+  "response_fields": [
+
+  ],
+  "requests": [
+    {
+      "request_method": "POST",
+      "request_path": "/v1/users/sign_in",
+      "request_body": "email=user2%40example.com&password=123456",
+      "request_headers": {
+      },
+      "request_query_parameters": {
+      },
+      "request_content_type": "application/x-www-form-urlencoded",
+      "response_status": 201,
+      "response_status_text": "Created",
+      "response_body": "{\"user\":{\"id\":2,\"authentication_token\":\"hS-Wnfqrszhbgy_C8kEW\",\"email\":\"user2@example.com\"}}",
```

Comment:
I've created issue for that https://github.com/fs/rails-base-api/issues/179, thank you!